doc/en/components/navigation-drawer.md

```diff
@@ -0,0 +1,237 @@
+# $Platform$ Navigation Drawer Overview
+
+The Infragistics $Platform$ Navigation Drawer is a WebComponent that provides side navigation. It can be expanded or collapsed within the content. A mini version provides quick access to navigation even when closed. Its content is completely customizable while also providing default menu item styling.
+
+> [!NOTE]
```

Comment:
Remove this note. We will have to add it in the general getting started topic because it's applicable to all components.